reframe/frontend/cli.py

```diff
@@ -210,7 +210,8 @@ def main():
 
     # Configure logging
     try:
-        logging.configure_logging(settings.logging_config)
+        logging.configure_logging(settings.logging_config,
+                                  settings.perf_logging_config)
```

Comment:
This will through an `AttributeError` if somebody tries to use this version with an old configuration file. I could perhaps catch it and issue a warning message. It would not be a problem if I was passing it as `None`, if the attribute didn't exist, but this would silently make the framework not to produce performance logs.